includes/html/forms/create-alert-item.inc.php

```diff
@@ -28,11 +28,8 @@
 $mute = $_POST['mute'];
 $invert = $_POST['invert'];
 $name = strip_tages($_POST['name']);
```

Comment:
This looks like a mistake. should be `$name = strip_tags($_POST['name']);` i suppose

Reply:
(i know it is not your code, but as you are around :D )